src/ServiceControl/ExternalIntegrations/IntegrationEventSender.cs

```diff
@@ -0,0 +1,44 @@
+namespace ServiceControl.ExternalIntegrations
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Logging;
+
+    public class IntegrationEventSender
+    {
+        static ILog log = LogManager.GetLogger<IntegrationEventSender>();
+
+        string[] integrationForwardQueues;
+        IBus bus;
+
+        public IntegrationEventSender(IBus bus, string[] integrationForwardQueues)
+        {
+            this.integrationForwardQueues = integrationForwardQueues;
+            this.bus = bus;
+        }
+
+        public void Send(object evnt)
+        {
+            foreach (var queue in integrationForwardQueues)
+            {
+                try
+                {
+                    bus.Send(Address.Parse(queue), evnt);
```

Comment:
Is there an issue with sending an event (rather than publishing it)? Do we need to disable the best practices analyzer (which I wouldn't be opposed to in SC anyway).